examples/with-indent/pages/find.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react'
+import { audit } from '@indent/audit'
+import Page from '../components/Page'
+
+export default class extends Component {
+  state = { sessionId: '' }
+
+  static getInitialProps = req => {
+    audit.write({
+      event: 'server_side_event',
```

Comment:
If you want to showcase server side events let's use API Routes, but not an empty `getInitialProps` like in this case.